external-crates/move/Cargo.toml

```diff
@@ -3,6 +3,9 @@ resolver = "2"
 
 members = [
     "crates/*",
+    "move-execution/v0/crates/*",
+    "move-execution/v1/crates/*",
+    "move-execution/v2/crates/*",
```

Comment:
Automating this is going to be ...tricky. The cut tool figures out which packages should be included or excluded from the root workspace here:

https://github.com/MystenLabs/sui/blob/127f2cb31699ebddbe1ea4935d264a235ca2e9ca/sui-execution/cut/src/plan.rs#L406

But it doesn't currently understand how to handle multiple workspaces, ...or glob patterns. One thing you could do is piggyback off the logic we have in the wrapper script, that updates TOML files when a new cut is introduced, here:

https://github.com/MystenLabs/sui/blob/127f2cb31699ebddbe1ea4935d264a235ca2e9ca/scripts/execution_layer.py#L420

This looks for comments in TOML files that contain "$CUT" and adds a line above them with "$CUT" replaced with the cut name, see e.g.

https://github.com/MystenLabs/sui/blob/127f2cb31699ebddbe1ea4935d264a235ca2e9ca/sui-execution/Cargo.toml#L18-L32

This is maybe a little more brittle, but I think it's the fastest way to get unblocked without losing the automation support.